vlm/src/main/scala/geotrellis/contrib/vlm/avro/GeoTrellisDataPath.scala

```diff
@@ -0,0 +1,14 @@
+package geotrellis.contrib.vlm.avro
+
+import geotrellis.contrib.vlm.DataPath
+
+
+case class GeoTrellisDataPath(val path: String) extends DataPath {
```

Comment:
```scala
val rs: RasterSource = RasterSource("gt+s3://azavea-datahub/catalog") 
// how do I specify the Layer name or zoom? I don't have enough information here.
```


## Option 1
Try to parse out the parameters from path
`gt+s3://azavea-datahub/catalog/nlcd` -> pick the best zoom level
`gt+s3://azavea-datahub/catalog/nlcd/12` -> pick the zoom level I told you

## Option 2
Use the query parameters
`gt+s3://azavea-datahub/catalog?layer=nlcd`
`gt+s3://azavea-datahub/catalog?layer=nlcd&zoom=12`